forest/cache_map/cache_map_manger.go

```diff
@@ -0,0 +1,112 @@
+package cachemap
```

Comment:
I think the filename is misspelled.  I am also confused what "man[a]ger" here means, it looks like a cache.